iMX8Pkg/iMX8CommonDsc.inc

```diff
@@ -622,7 +622,8 @@
   gEfiSecurityPkgTokenSpaceGuid.PcdTpmInstanceGuid|{ 0x5a, 0xf2, 0x6b, 0x28, 0xc3, 0xc2, 0x8c, 0x40, 0xb3, 0xb4, 0x25, 0xe6, 0x75, 0x8b, 0x73, 0x17 }
 
 [PcdsDynamicExDefault.common.DEFAULT]
-  gEfiSecurityPkgTokenSpaceGuid.PcdTcg2HashAlgorithmBitmap|0x0
+  # Tcg2Dxe registers Sha1 and Sha256
+  gEfiSecurityPkgTokenSpaceGuid.PcdTcg2HashAlgorithmBitmap|0x3
 !endif
```

Comment:
Once the switch is made to the updated fTPM this will need to also include SHA386 and SHA512 (Need to add the extra libraries to the list in SecurityPkg/Tcg/Tcg2Dxe/Tcg2Dxe.inf { … on line 799 as well)